CHANGELOG.md

```diff
@@ -70,6 +70,7 @@ for further information about branching and tagging conventions.
 - Invalid start date for sales that are already started
 - Fixed a bug with impossible order cancelling if user has insufficient balance
 - Fixed a bug with chart line rendering
+- Fixed a bug with cannot invest if asset balance is 0 on `Invest` form
```

Comment:
Don't add a `Fixed` at the sentencing front. You already have a `Fixed` title above. Also, this message describes nothing. That's was a bug. Which bug, how does component works right now?

I think you need to move PR describe under the `Changed` title and write something like:
`Warning/message about insufficient balance in the "Invest" form`